client/src/components/Tickets/TicketModal.js

```diff
@@ -180,10 +205,11 @@ TicketHistoryCard.propTypes = {
   created: PropTypes.instanceOf(Date).isRequired,
   creator: PropTypes.string.isRequired,
   ticketCreator: PropTypes.bool.isRequired,
-  content: PropTypes.string.isRequired
+  content: PropTypes.string.isRequired,
+  attachments: PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
After looking again, this is an array of arrays
```suggestion
  attachments: PropTypes.arrayOf(PropTypes.array).isRequired
```